pkg/koordlet/metricsadvisor/collectors/coldmemoryresource/cold_page_collector.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package coldmemoryresource
+
+import (
+	"go.uber.org/atomic"
+
+	"github.com/koordinator-sh/koordinator/pkg/koordlet/metricsadvisor/framework"
+	"github.com/koordinator-sh/koordinator/pkg/koordlet/util/system"
+)
+
+const (
+	CollectorName = "ColdPageCollector"
+)
+
+type nonColdPageCollector struct {
+}
+
+func New(opt *framework.Options) framework.Collector {
+	// check whether support kidled cold page info collector
+	if system.IsKidledStart() {
+		return &kidledcoldPageCollector{
+			collectInterval: opt.Config.CollectResUsedInterval,
```

Comment:
The interval `CollectResUsedInterval` is 1 second for global use. Maybe we should define an individual interval for the cold page collection.